src/screens/Exchange/Exchange.js

```diff
@@ -349,41 +349,53 @@ function SelectorInputTemplate(locals) {
       inputRef,
       onSelectorOpen,
     },
+    value,
   } = locals;
+  const { selector = {} } = value;
```

Comment:
Just thinking maybe use lodash.get here for `value` and use `{}` fallback as form input tend to produce undefined / null values, what do you think?